feature/policy_forwarding/encapsulation/otg_tests/inner_ttl_ipv4/README.md

```diff
@@ -1,94 +1,389 @@
+    inner IP TTL set to `unmatched_ip_ttl` - 1.
 
-Canonical OpenConfig for policy forwarding, matching IP prefix and TTL = 1 with action
-set inner packet TTL = 1.
+### TE-1.11.2 Rewrite the ingress innner packet TTL = 1, if the incoming TTL = 1
+for IPv6 traffic.
+
+ATE action:
+*   Generate total 10,000,000 **IPv6 packets** from ATE:Port1 and ATE:Port2
+    with:
+    *   Source IP from random addresses in `matched_ipv6_src_net` to destination
+        IP addresses in `ipv6_dst_net` addresses.
+    *   Use 512 bytes frame size.
+    *   Set TTL of all packets to `matched_ip_ttl`.
+
+*   Generate total 10,000,000 **IPv6 packets** from ATE:Port1 and ATE:Port2
+    with:
+    *   Source IP from random addresses in `unmatched_ipv6_src_net` to
+        destination IP addresses in `ipv6_dst_net` addresses.
+    *   Use 512 bytes frame size.
+    *   Set TTL of all packets to `unmatched_ip_ttl`.
+
+Verify:
+*   The total packet count of traffic sent from ATE:Port1 and ATE:Port2 should
+    be equal to the sum of all packets received on ATE:Port3 and ATE:Port4.
+*   All packets received on ATE:Port3 and ATE:Port4 are encapsulated with
+    MPLSoGRE IPv4 with:
+    *   MPLS label: `mpls_label`
+    *   Source IPv4 address: `ipv4_tunnel_src`
+    *   Destination IPv4 address: `ipv4_tunnel_dst_a` or `ipv4_tunnel_dst_b`
+    *   Tunnel IP TTL: `tunnel_ip_ttl`
+*   Packets with source IPv6 addresses from subnet `matched_ipv6_src_net` has
+    inner IP TTL set to `rewritten_ip_ttl`.
+*   Packets with source IPv6 addresses from subnet `unmatched_ipv6_src_net` has
+    inner IP TTL set to `unmatched_ip_ttl` - 1.
+
+## Canonical OC
 
 ```json
 {
-  "openconfig-network-instance": {
-    "network-instances": [
+  "network-instances": {
+    "network-instance": [
       {
-        "afts": {
-          "policy-forwarding": {
-            "policies": [
+        "name": "test_vrf",
+        "config": {
+          "name": "test_vrf"
+        },
+        "policy-forwarding": {
+          "policies": {
+            "policy": [
               {
+                "policy-id": "retain ttl",
                 "config": {
                   "policy-id": "retain ttl",
-                  "type": "PBR_POLICY"
+                  "type": "MATCH_ACTION"
                 },
-                "policy": "retain ttl",
-                "rules": [
-                  {
-                    "config": {
+                "rules": {
+                  "rule": [
+                    {
                       "sequence-id": 1,
+                      "config": {
+                        "sequence-id": 1
+                      },
+                      "ipv4": {
+                        "config": {
+                          "hop-limit": 1
+                        }
+                      },
+                      "action": {
+                        "config": {
+                          "next-hop-group": "NHG-1",
+                          "ip-ttl": 1
```

Comment:
Add text just before the JSON like:

`TODO: add  ip-ttl  policy-forwarding action to OpenConfig Public data models, pending https://github.com/openconfig/public/pull/1313`.